<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckoutDialog</class>
 <widget class="QDialog" name="CheckoutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="cartSummaryList">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalPriceLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="creditCardLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Card Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name for the order</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLine">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Special Requests</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="specialRequestLineEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>141</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
